### ISSUE TITLE
Alterac Valley: Bowmen Despawning from Tower or Bunker That is Not Destroyed

### ISSUE THREAD

**Issue by turetafr** [LightsHope/server#514]
Capping an enemies base graveyard (Aid Station / Relief Hut) despawns ALL of the enemies tower/bunker bowmen.  This is a bug. Bowmen should not despawn unless their tower is destroyed.

**Comment by Ripper007** [comment]
This is intended. Before it was worse, the archers in horde would despawn when alliance would cap Frostwolf Graveyard. So they made it same for both factions.

**Comment by Cabro** [comment]
Intended or not, is a bug.

**Comment by turetafr** [comment]
Archers do not despawn when Frostwolf graveyard is capped.  There is no version of WoW where archers are supposed to despawn until the tower or bunker is burned.

**Comment by Ripper007** [comment]
The FW GY alliance cap  archers despawning bug was  on Nostalrius servers .. the pvp server had it, the pve server had it, then when transfer to Elysium happend and they integrated the Nos core, the issue was present on all the servers. At one point devs applied some fixes to the AV including this one and to make it "fair" they also linked to despaning to be the same for both factions.

**Comment by turetafr** [comment]
This is wrong.  Bowmen should not be linked to a graveyard. They are linked to the tower they are protecting.  Bowmen should only despawn if their tower is destroyed.  Anything other than this is not Blizlike.  Go play retail and you will see. You won't find any mention of it in patches because it was never bugged in retail. If Nost had this bug then it was unblizlike from the start.

**Comment by turetafr** [comment]
This is wrong.  Bowmen should not be linked to a graveyard. They are linked to the tower they are protecting.  Bowmen should only despawn if their tower is destroyed.  Anything other than this is not Blizlike.  Go play retail and you will see. You won't find any mention of it in patches because it was never bugged in retail. If Nost had this bug then it was unblizlike from the start.

**Comment by turetafr** [comment]
Basically what they did on Nost is make the bug worse by making it bugged on both sides, which makes it even more UnBlizlike.

**Comment by turetafr** [comment]
Bowmen should not be linked to a graveyard. They should be linked to the tower they are protecting. Bowmen defending a specific tower should only despawn if their tower is destroyed. Anything other than this is not Blizlike. Nothing in the mechanics of Alterac Valley ever allowed you to despawn all bowmen without burning a single tower or bunker.  Go play retail and you will see. You won't find any mention of it in patches because it was never bugged in retail. If Nost had this bug then it was unblizlike from the start, and making it "even" on both sides is essentially the equivalent of adding another bug to the game intentionally, which is even more unblizlike. Pleae fix.

**Comment by turetafr** [comment]
Bowmen should not be linked to a graveyard. They should be linked to the tower they are protecting. Bowmen defending a specific tower should only despawn if their tower is destroyed. Anything other than this is not Blizlike. Nothing in the mechanics of Alterac Valley ever allowed you to despawn all bowmen without burning a single tower or bunker. Go play retail and you will see. You won't find any mention of it in patches because it was never bugged in retail. If Nost had this bug then someone intentionally added this and it was unblizlike from the start, and making it "even" on both sides is essentially the equivalent of adding another bug to the game intentionally. It should be fixed properly.
